apps/site/assets/ts/stop/components/StopPageHeaderRedesign.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from "react";
+import { Stop } from "../../__v3api";
+import { TypedRoutes } from "./__stop";
+import StopFeatures from "./StopFeatures";
+import { typedRoutesHasBusRoute } from "../../helpers/routes";
+
+const StopPageHeaderRedesign = ({
+  stop,
+  typedRoutes
+}: {
+  stop: Stop | undefined;
+  typedRoutes: TypedRoutes[] | undefined;
+}): ReactElement<HTMLElement> => {
+  return (
+    <div className="u-bg--gray-bordered-background">
+      <header className="d-flex justify-content-space-between container">
+        <div className="d-flex">
+          <h3>{stop && stop.name}</h3>
```

Comment:
This might be something we want to bring in front of Anna.
I can't figure out how to mimic that behavior off the top of my head.
Flex box will wrap the icons on a separate line from the text on small screens.
Putting the icons in the same element as the text will only wrap the text with the icons not moving
![Screen Shot 2023-03-10 at 3 14 41 PM](https://user-images.githubusercontent.com/2679229/224430026-1481502b-6ea7-4c7e-9433-82dddbdc30bd.png)

Reply:
My guess is you'd have to either put the icons inside the `<h3>` and/or override the `<h3>`'s default `display: block` to `inline-block` or `inline`.

Reply:
Had a conversation with Anna.  Decided to stack the stop number and the icons as per the [figma](https://www.figma.com/file/7AJep6p9vIdg3bRVKSqEjq/%F0%9F%93%A6-MBTA.com-Design-%3C%3E-Dev-Handoff?node-id=934-10108&t=thDBccVLfR59u5wY-0)
![Screen Shot 2023-03-20 at 9 16 55 AM](https://user-images.githubusercontent.com/2679229/226368092-5a6576a1-60aa-41c3-86f0-024aed9231b9.png)